pkg/precompiles/warp.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package precompiles
+
+import (
+	_ "embed"
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/pkg/contract"
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+func WarpPrecompileGetBlockchainID(
+	rpcURL string,
+) (ids.ID, error) {
+	out, err := contract.CallToMethod(
+		rpcURL,
+		WarpPrecompile,
+		"getBlockchainID()->(bytes32)",
+	)
+	if err != nil {
+		return ids.Empty, err
+	}
+	received, b := out[0].([32]byte)
```

Comment:
Do we need to validate the out array here before we call this?

Reply:
For the second question, I could be rusty on Go. More so, I am looking to confirm that this conversion fails, if it'll throw an uncaught error

Reply:
First question. We are counting that the underlying implementation, if does not error, returns out
with  the appropriate size. This is the current semantic.
That say, better to check for len!

Reply:
Second question. The type conversion is safe if you also get the b boolean. I mean, there is no uncaught error.

If you do
```
received := out[0].([32]byte)
```
you can have uncaught error.

If you do
```
received, b := out[0].([32]byte)
```
you will have b==false in conversion error